std_msgs/Header       header

uint8[]               idlist
uint16[]               snlist
uint16[]              ch_val
uint32[]             raw_times
float32[]            rolled_times
float32[]            sync_times

float32               raw_rate
float32               gate_rate
